planning/behavior_path_planner/src/scene_module/pull_over/pull_over_module.cpp

```diff
@@ -307,6 +313,49 @@ bool PullOverModule::isLongEnoughToParkingStart(
   return *dist_to_parking_start_pose > current_to_stop_distance;
 }
 
+bool PullOverModule::checkCollisionWtihLongitudinalDistance(
+  const Pose & ego_pose, const PredictedObjects & dynamic_objects) const
+{
+  if (parameters_.use_occupancy_grid) {
+    bool check_out_of_range = false;
+    const double offset = std::max(
+      parameters_.goal_to_obj_margin - parameters_.occupancy_grid_collision_check_margin, 0.0);
```

Comment:
I am confused by the following 
- If there is no recognized object, should we consider `goal_to_obj_margin` as `goal_to_point_margin` ?
- >goal_to_object_margin : 0.0
  the distance between ego to object is 1.0(same to object_recognition_collision_check_margin)
  
  For that parameter value, the offset is 0. Is that correct?

Reply:
thank you for your comments.

> If there is no recognized object, should we consider goal_to_obj_margin as goal_to_point_margin ?

sorry, what does `goal_to_point_margin` mean?

> For that parameter value, the offset is 0. Is that correct?

Yes, correct. In this case  `goal_to_obj_margin` does not work, only the usual collision check margins like `object_recognition_collision_check_margin` and `object_recognition_collision_check_margin` are used.

Reply:
> sorry, what does goal_to_point_margin mean?

` goal_to_obj_margin` sounds like a parameter for "recognized object" to me. 
But am I correct that you are including it as a "pointcloud" as well?

Reply:
yes, It also includes pointcloud(occupancy_grid).

Reply:
> goal_to_obj_margin sounds like a parameter for "recognized object" to me.

hmm, `goal_to_obstacle_margin` is better?

Reply:
Thank you I fully understand!

Reply:
> goal_to_obstacle_margin is better?

Nice! Whichever is good for me

Reply:
(Maybe previously `goal_to_obj_margin` is used for occupancy_grid, and if no body doubt it, it may be understandable)

Reply:
yes previously it is used for occupancy_grid. But I will change the name.

Reply:
Excellent!

Reply:
fixed in 
https://github.com/autowarefoundation/autoware.universe/pull/1796/commits/6907b0893e8374eba408eed26668d4cda01c6486

Reply:
I also update launcher's one https://github.com/tier4/autoware_launch/pull/470